app/src/main/java/com/google/samples/apps/sunflower/PlantDetailFragment.kt

```diff
@@ -96,6 +96,7 @@ class PlantDetailFragment : Fragment() {
                             addFlags(Intent.FLAG_ACTIVITY_NEW_DOCUMENT or Intent.FLAG_ACTIVITY_MULTIPLE_TASK)
                         } else {
                             // Else, we will use the old CLEAR_WHEN_TASK_RESET flag
+                            @Suppress("DEPRECATION")
```

Comment:
Can you move this to line 84?  It's easier to notice that way.

Reply:
done